src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -632,5 +632,22 @@ public void ObjectDump()
                 view.ObjectDump();
             }
         }
+
+        /// <summary>
+        /// Search through this View's hierarchy for a View with the given unique ID.
+        /// </summary>
+        /// <param name="id">The ID of the View to find.</param>
+        /// <remarks>Hidden-API</remarks>
+        /// <returns>A handle to the View if found, or an empty handle if not.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View FindDescendantByID(uint id)
+        {
+            //to fix memory leak issue, match the handle count with native side.
+            IntPtr cPtr = Interop.Actor.Actor_FindChildById(swigCPtr, id);
+            View ret = this.GetInstanceSafely<View>(cPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
```

Comment:
You do better to use existed method, or I recommend to remove `internal FindChildById` and replace to this API
```suggestion
            return FindChildById(id);
```

Reply:
Duplicated code and APIs

https://github.com/Samsung/TizenFX/blob/7a14e7532a6622108bc0119764a964475f371773/src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs#L499-L506

https://github.com/Samsung/TizenFX/blob/7a14e7532a6622108bc0119764a964475f371773/src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs#L508-L511